homestead.local_settings/local_settings.py

```diff
@@ -7,13 +7,16 @@
 # Otherwise no rights are granted except for those provided to you by
 # Metaswitch Networks in a separate written agreement.
 
+import tempfile
```

Comment:
What's this all about?

Reply:
Will delete, was supposed to create random filename for the temp file, didn't work as the UNIX filename needs to be certain format